### PR TITLE
[C#] Update after.md document for dates exercise

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: languages/csharp/exercises/concept/dates/.docs/after.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # After
 
-- Interpolated strings support [specifying format strings][docs.microsoft.com_specify-a-format-string-for-an-interpolation-expression] for their interpolated values.
+In C# dates are represented as `DateTime` instances, which contains both date _and_ time information. Dates can be compared using the default comparison operators (`<`, `>`, etc.). The current date (and time) can be retrieved through the static `DateTime.Now` property.
```
Comment (robkeim) [inline languages/csharp/exercises/concept/dates/.docs/after.md]: Is it worth mentioning that `DateTime.Now` is in the local timezone of the computer (even though timezones are out of scope for this question)?

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/dates/.docs/after.md]: I left it out on purpose as to not confuse the user with time zone issues. Would you rather add it?

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/dates/.docs/after.md]: p.s. I merged so that can we PR any changes later.

Comment (robkeim) [reply languages/csharp/exercises/concept/dates/.docs/after.md]: I'm fine not having timezones as a part of this exercise. It's a difficult concept and might be too much to just throw in a one line sentence in the after.

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/dates/.docs/after.md]: Great!

Comment (ErikSchierboom) [reply languages/csharp/exercises/concept/dates/.docs/after.md]: p.s. I just found out that the dates and time zones Concepts were missing from our list of Concepts: https://github.com/exercism/v3/pull/788

**Pull request merged**